core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -138,12 +138,15 @@ public void mastersStartedCallback() {
   public LockResource lockShared() throws InterruptedException {
     if (LOG.isDebugEnabled()) {
       LOG.debug("Thread-{} entered lockShared().", ThreadUtils.getCurrentThreadIdentifier());
+    }
+    if (LOG.isInfoEnabled()) {
       final int readLockCount = mStateLock.getReadLockCount();
-      if (readLockCount > READ_LOCK_COUNT_HIGH) {
+      if (readLockCount > READ_LOCK_COUNT_HIGH && LOG.isInfoEnabled()) {
```

Comment:
nit: `LOG.isInfoEnabled` is always true here right?

Reply:
oops. fixed.